src/_posts/2017-02-01-graduation-media-recap.md

```diff
@@ -33,7 +33,6 @@ include:
 * Datanami: "[Google Lauds Outside Influence on Apache Beam](https://www.datanami.com/2017/01/10/google-lauds-outside-influence-apache-beam/)" by Alex Woodie.
 * InfoWorld / JavaWorld: "[Apache Beam unifies batch and streaming for big data](http://www.infoworld.com/article/3156598/big-data/apache-beam-unifies-batch-and-streaming-for-big-data.html)" by Serdar Yegulalp, and republished in [JavaWorld](http://www.javaworld.com/article/3156598/big-data/apache-beam-unifies-batch-and-streaming-for-big-data.html).
 * JAXenter: "[In a way, Apache Beam is the glue that connects many big data systems together](https://jaxenter.com/apache-beam-interview-131314.html)" by Kypriani Sinaris.
-* OStatic: "[Apache Beam Unifies Batch and Streaming Data Processing](http://ostatic.com/blog/apache-beam-unifies-batch-and-streaming-data-processing)" by Sam Dean.
```

Comment:
Same comment as before -- I'd prefer if the content of the blog doesn't change; just comment out the link, not the entire bullet.